0200-background.Rmd

```diff
@@ -155,6 +153,26 @@ Detailed reviews of Morice River watershed fisheries can be found in @bustard_sc
 
 <br>
 
+### Zymoetz River
+
+Within the Zymoetz Watershed, there are many areas with high fishery values. Steelhead are the most extensively documented fish species in the Zymoetz River watershed. Adults enter the river from JUly to November and then go on to spawn the following year in late spring to early summer. The Zymoetz River is a relatively steep system. Two canyons are located 6.4 and 19.6 kilometers upstream of the Skeena River confluence. These canyons make access to the Zymoetz difficult for pink and chum salmon [@gottesfeld_etal2002ConservingSkeena]. The Zymoetz River is renowned for its aggressive steelhead that have been known to take flies or lures. There is a 50km stretch upstream of Limonite Creek where the Zymoetz River is a designated Class I water. This section is remote and offers high quality fishing opportunities for anglers [@Zymoetz1_AMP.pdf].

```

Comment:
What does Class 1 water mean?  Requires a specific permit and guide for non-residents or something (I htink that is Class 2).  I actually don't know so we shoud define. I'll make issue